File: NetFramework/DashboardLayout/VB/DashboardLayoutDemo/DemoControls/FlexGrid.vb
Imports C1.Win.FlexGrid

Namespace DashboardDemo.Controls
    Partial Public Class DemoFlexGrid
        Inherits C1FlexGrid
        Implements IDemo

        Public Sub Init() Implements IDemo.Init
            DataSource = SampleHelper.Instance.GetSourceTable("Sales", Resources.Sales)
        End Sub
    End Class
End Namespace

File: NetFramework/DashboardLayout/CS/DashboardLayoutDemo/DemoControls/FlexGrid.cs
using C1.Win.C1FlexGrid;

namespace DashboardLayoutDemo.Controls
{
    public partial class DemoFlexGrid : C1FlexGrid, IDemo
    {
        public void Init() =>  DataSource = SampleHelper.Instance.GetSourceTable("Sales", Properties.Resources.Sales);
    }
}
